python/pacemaker/_cts/watcher.py

```diff
@@ -218,9 +238,6 @@ def __init__(self, log, regexes, hosts, kind, name="Anon", timeout=10, silent=Fa
         if not self.hosts:
             raise ValueError("LogWatcher requires hosts argument")
 
-        if not self.kind:
-            raise ValueError("LogWatcher requires kind argument")
-
```

Comment:
technically someone could explicitly pass something like kind=None but I suppose we can ignore that

Reply:
Yeah, I thought about that.  But that someone would have to be us, and I figure that'll trigger problems elsewhere, so the result is probably the same - a traceback that we'll have to debug.